mantidimaging/core/io/loader/img_loader.py

```diff
@@ -95,6 +103,8 @@ def _do_files_load_seq(self, data: pu.SharedArray, files: list[str]) -> pu.Share
                 except OSError as exc:
                     raise RuntimeError(f"Could not load file {in_file}. Error details: {exc}") from exc
 
+        LOG.info(f"Loaded {len(files)} files (name={os.path.basename(files[0])}, format={self.img_format}, "
+                 f"total size={total_size / (1024 * 1024):.2f} MB, dtype={dtype}, shape={shape})")
```

Comment:
shape and data type can be used here, rather than needing to get them in the loop.
```
        LOG.info(f"Loaded {len(files)} files (name={os.path.basename(files[0])}, format={self.img_format}, "
                 f"total size={total_size / (1024 * 1024):.2f} MB, dtype={data.array.dtype}, shape={data.array.shape})")
```